_gsocproposals/2020/proposal_IRIS-HEPskyhookdm.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Extend SkyhookDM programmable object storage with statistics, sort/aggregate, or data compaction functions.
+layout: gsoc_proposal
+project: IRIS-HEP
+year: 2020
+organization: CROSS/UC Santa Cruz
```

Comment:
You will have to add the organization as well

Reply:
The *organization:* field name needs to be compact and without special characters, e.g CROSS-UCSC and has to match the one in _gsocorgs/2020/cross-ucsc.md (note the file name for the org file). the logo file name used in cross-ucsc.md should be then called CROSS-UCSC-logo.png (the logo file needs to be put in the images folder

Reply:
done.